src/editor/component_panels/signals_panel.cpp

```diff
@@ -106,8 +106,12 @@ Dictionary OrchestratorScriptSignalsComponentPanel::_handle_drag_data(const Vect
     TreeItem* selected = _tree->get_selected();
     if (selected)
     {
-        data["type"] = "signal";
-        data["signals"] = Array::make(selected->get_text(0));
+        Ref<OScriptSignal> signal = _orchestration->find_custom_signal(StringName(selected->get_text(0)));
```

Comment:
```suggestion
        Ref<OScriptSignal> signal = _orchestration->find_custom_signal(selected->get_text(0));
```
Consider changing the argument type from `StringName` to `String` for simplicity.